src/plugins/auto_batch/src/compiled_model.cpp

```diff
@@ -179,13 +180,6 @@ ov::Any CompiledModel::get_property(const std::string& name) const {
     if (it != m_config.end()) {
         return it->second;
     } else {
-        // find config key among networks config keys
-        auto modelSupportedProperties = m_compiled_model_without_batch->get_property(ov::supported_properties.name());
```

Comment:
Why did you remove this code?

Reply:
Didn't remove this, just move this to:
https://github.com/openvinotoolkit/openvino/blob/0555a492313a666c1a7d7e8cef4c0db26407ccdc/src/plugins/auto_batch/src/compiled_model.cpp#L222-L230
The original is a bug, when call get_property for `ov::supported_properties`, which will return the properties from HW plugin compiler model not for auto-batch plugin compiler model. So add the following code:
https://github.com/openvinotoolkit/openvino/blob/0555a492313a666c1a7d7e8cef4c0db26407ccdc/src/plugins/auto_batch/src/compiled_model.cpp#L214-L222